src/game-engine/GameEngine.ts

```diff
@@ -52,11 +51,11 @@ export default class GameEngine {
     const nextPledge = new PreFundSetup(channel, 0, balances, 0, stake);
     const message = this.channelWallet.sign(nextPledge.toHex());
 
-    const appState = new ApplicationStatesA.ReadyToSendPreFundSetup0({
+    const appState = new ApplicationStatesA.ReadyToSendPreFundSetupA({
       channel,
       stake,
       balances: balances,
-      signedPreFundSetup0Message: message.toHex(),
+      signedPreFundSetupAMessage: message.toHex(),
```

Comment:
Wondering if it might just be worth changing all of these to just be “message”? The type of message can be determined from the state. I’m not sure why I did it like this in the first place ...

Reply:
I'm inclined to agree